src/qibo/backends/numpy.py

```diff
@@ -93,7 +93,7 @@ def identity_density_matrix(self, nqubits, normalize: bool = True):
 
     def plus_state(self, nqubits):
         state = self.np.ones(2**nqubits, dtype=self.dtype)
-        state /= self.np.sqrt(2**nqubits)
+        state /= self.np.sqrt(self.cast(2**nqubits))
```

Comment:
I'd prefer something like 

```suggestion
        state /= math.sqrt(2**nqubits)
```